src/settings/routes/PickListSettings.js

```diff
@@ -17,13 +17,20 @@ export default class PickListSettings extends React.Component {
     this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
   }
 
+  suppressDelete = (category) => {
+    const { internal, values = [] } = category;
+
+    return !(!internal && !values.length);
```

Comment:
[Same as in Licenses](https://github.com/folio-org/ui-licenses/pull/327#discussion_r400417956)